logger.js

```diff
@@ -51,29 +64,31 @@ function pinoLogger (opts, stream) {
     var log = this.log
     var responseTime = Date.now() - this[startTime]
     var level = customLogLevel ? customLogLevel(this, err) : useLevel
+    var payload = {}
```

Comment:
Please define all properties in one statement, as it will create a single shape.

Reply:
```
var payload = {
  [resKey]: this,
  [responseTimeKey]: responseTime,
  [errKey]: error
}
```

As the attribute names are now dynamic, we'd need computed property syntax to do this in one statement, but this is only compatible with node > 4. Is that acceptable?

Reply:
yes it is.

Reply:
Great, done.